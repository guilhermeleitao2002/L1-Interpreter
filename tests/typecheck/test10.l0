type Point = struct { x: int, y: int };
let p = { x = "wrong", y = 10 };
p;;