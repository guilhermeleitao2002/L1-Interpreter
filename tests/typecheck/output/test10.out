Type Error:
TypeError: Cannot assign string to int in struct field